modular_skyrat/modules/colony_fabricator/code/design_datums/organic_printer_designs/resources.dm

```diff
@@ -0,0 +1,10 @@
+/datum/design/organic_plastic
+	name = "Plastic Sheet"
+	id = "oganic_plastic"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 25)
```

Comment:
I'm still not entirely sold on this, I don't know, it feels a bit too easy to get plastic. I'll have to think over it some more.

Reply:
Consider that the main source of plastic is someone going to a chem machine, pressing like five buttons and having as much plastic as they want for practically free forever. This is frankly worse than using the intended way to get it (outside of the fact cargo can order 50 sheets of plastic for like 800 credits I think)